packages/backend/src/models/HuggingFaceModelHandler.ts

```diff
@@ -129,7 +129,7 @@ export class HuggingFaceModelHandler extends ModelHandler {
               if (model) {
                 model.model.file = {
                   path: revision.path,
-                  file: revision.path,
+                  file: '',
```

Comment:
By setting the file as empty, the function `ModelsManager.isModelOnDisk` does not work anymore, as it was relying on this field to check if the model has been downloaded

Reply:
I suppose that `.` or `*` would be meaningful. I'm not sure if it would hurt somewhere else

Reply:
> By setting the file as empty, the function `ModelsManager.isModelOnDisk` does not work anymore, as it was relying on this field to check if the model has been downloaded

It is based on the upper file field and not this one

Reply:
> I suppose that `.` or `*` would be meaningful. I'm not sure if it would hurt somewhere else

It used an empty string because the path is shown when deleting the model from disk thus that would seems a little strange if an extra . and * would be displayed